From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Wed, 15 Aug 2018 17:13:57 +0200
Subject: Revert "main: Remove GNOME_DESKTOP_SESSION_ID envvar"

This reverts commit 00e0e6226371d53f651cc881e74c0543192c94a8.

Keep this for compatibility reasons, many apps still depends on this

This is Debian-specific - upstream have chosen to remove this compatibility but
we'll keep it.

Forwarded: not-needed
---
 gnome-session/main.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/gnome-session/main.c b/gnome-session/main.c
index 443bc04..d30884b 100644
--- a/gnome-session/main.c
+++ b/gnome-session/main.c
@@ -602,6 +602,11 @@ main (int argc, char **argv)
                 g_free (ibus_path);
         }
 
+        /* Some third-party programs rely on GNOME_DESKTOP_SESSION_ID to
+         * detect if GNOME is running. We keep this for compatibility reasons.
+         */
+        gsm_util_setenv ("GNOME_DESKTOP_SESSION_ID", "this-is-deprecated");
+
         /* We want to use the GNOME menus which has the designed categories.
          */
         gsm_util_setenv ("XDG_MENU_PREFIX", "gnome-");
